src/Form/Form.js

```diff
@@ -59,51 +63,75 @@ class Form {
         this.categorizeInputs()
     }
 
+    hasFocus () {
+        return this.form.contains(document.activeElement)
+    }
```

Comment:
We now use `hasFocus` to check that the form is focused when we detect an `Enter` keypress and fire the `submitHandler`.